diff --git a/dist/module-evaluator.js b/dist/module-evaluator.js
--- a/dist/module-evaluator.js
+++ b/dist/module-evaluator.js
@@ -339,7 +339,7 @@
 		defaultExport: mod
 	};
 	let defaultExport = "default" in mod ? mod.default : mod;
-	if (!isPrimitive(defaultExport) && "__esModule" in defaultExport) {
+	if (!isPrimitive(defaultExport) && defaultExport.__esModule) {
 		mod = defaultExport;
 		if ("default" in defaultExport) defaultExport = defaultExport.default;
 	}
